packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1477,3 +1479,34 @@ async fn test_wallet_balance_api() {
         );
     }
 }
+
+#[tokio::test]
+async fn test_transaction_script_workflow() {
+    abigen!(
+        MyContract,
+        "packages/fuels-abigen-macro/tests/test_projects/contract_test/out/debug/contract_test-abi.json"
+    );
+
+    let wallet = launch_provider_and_get_single_wallet().await;
+    let client = &wallet.get_provider().unwrap().client;
+
+    let contract_id = Contract::deploy(
+        "tests/test_projects/contract_test/out/debug/contract_test.bin",
+        &wallet,
+        TxParameters::default(),
+    )
+    .await
+    .unwrap();
+
+    let contract_instance = MyContract::new(contract_id.to_string(), wallet.clone());
+
+    let call_handler = contract_instance.initialize_counter(42);
+
+    let script = Script::from_call(&call_handler.contract_call).await;
+    assert!(script.tx.is_script());
+
+    let receipts = script.call(client).await.unwrap();
+
+    let response = call_handler.build_response(receipts).unwrap();
+    assert_eq!(response.value, 42);
```

Comment:
This kind of workflow is worse from a user standpoint. We now need to write 3 lines just to get a response and receipts out.

Reply:
The old workflow is retained, notice nothing is changed for the old tests except for calling it `call_handler`. This is optional to satisfy #200.

Reply:
Oh ok, wasn't sure!